modules/services/pulseeffects.nix

```diff
@@ -0,0 +1,54 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.pulseeffects;
+
+  presetOpts = optionalString (cfg.preset != "") "--load-preset ${cfg.preset}";
+
+in {
+  meta.maintainers = [ maintainers.jonringer ];
+
+  options.services.pulseeffects = {
+    enable =
+      mkEnableOption "Pulseeffects daemon. Used for advanced audio mixing.";
+
+    preset = mkOption {
+      type = types.lines;
```

Comment:
Should this really be a multi-line string? The `presetOpts` seem to use it as a plain command line argument so newlines could cause issues?

Reply:
probably not. I think i was just trying to work initially, and this was the one that came to mind

Reply:
Then perhaps `types.str` might be a better choice.

Reply:
I changed it to exactly that :)